pegasus/sites.v3/code.org/public/ai/index.haml

```diff
@@ -130,7 +130,18 @@ theme: responsive_full_width
           %a.link-button{href: CDO.studio_url("/courses/self-paced-pl-aiml-2023")}
             =hoc_s(:call_to_action_self_paced_pl)
 
-= view :section_divider_line
+
+%section.bg-neutral-light
+  .wrapper{style: "display: flex; justify-content: space-between; align-items: center; flex-wrap: wrap"}
+    %figure.col-45
```

Comment:
Can you add a `margin-bottom: 1rem` here so there is some space on tablet and mobile?

Right now there's no space between the elements:
<img width="250" alt="Screenshot 2023-10-13 at 12 18 29 PM" src="https://github.com/code-dot-org/code-dot-org/assets/9256643/e899c669-0d77-441a-a629-44436778841f">